recipes/azure-common/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: azure-common
+  version: {{ version }}
+
+source:
+  fn: azure-common-{{ version }}.zip
+  url: https://pypi.python.org/packages/48/89/017df9bed55925de004076937ca168d3a03aa0250fbda7826ec3c0ac714f/azure-common-{{ version }}.zip
+  md5: 13471213f49400c5d483865c96558ab9
+  
+build:
+  script: pip install --no-deps .
```

Comment:
Normally, we use `python setup.py install --single-version-externally-managed --record=record.txt` for `setuptools`-based installs and `python setup.py install` for `distutils`-based install. Though there are a few places where `pip` ends up being used too.